helix-term/src/ui/completion.rs

```diff
@@ -27,38 +29,58 @@ impl menu::Item for CompletionItem {
         self.label.as_str()
     }
 
-    fn row(&self) -> menu::Row {
+    fn row(&self, theme: Option<&Theme>) -> menu::Row {
+        macro_rules! get_style {
+            ($id:literal) => {
+                theme
+                    .and_then(|theme| theme.try_get($id))
+                    .map_or(Style::default(), |mut style| {
+                        style.bg = None;
+                        style
+                            .remove_modifier(Modifier::BOLD)
+                            .add_modifier(Modifier::ITALIC)
+                    })
+            };
+        }
```

Comment:
This is going to expand into _a lot_ of code. Return `Some("function.method")` from the match statement, then do this call right afterwards

Reply:
You're right, this is better. Fixed.